src/core/PasswordGenerator.h

```diff
@@ -34,14 +34,16 @@ class PasswordGenerator
         UpperLetters      = 0x2,
         Numbers           = 0x4,
         SpecialCharacters = 0x8,
-        EASCII            = 0x10
+        EASCII            = 0x10,
+        DefaultCharset    = LowerLetters | UpperLetters | Numbers
     };
     Q_DECLARE_FLAGS(CharClasses, CharClass)
 
     enum GeneratorFlag
     {
         ExcludeLookAlike   = 0x1,
-        CharFromEveryGroup = 0x2
+        CharFromEveryGroup = 0x2,
+        DefaultFlags       = ExcludeLookAlike | CharFromEveryGroup
     };
```

Comment:
I think it makes sense to convert these two enums to enum classes.

Reply:
This is a pretty big change, maybe we can ship this specific one in 2.4.0 or a 2.3 minor

Reply:
It should be more or less an automatic refactor. You may have to add the class name at several locations where this enum is used without its full name, but if you encounter an instance where this creates problems, because both enums were mixed, you most certainly found a bug.

Reply:
The thing is that with `enum class` you don't have automatic casting `error: no match for ‘operator&’ (operand types are PasswordGenerator::CharClass’ and ‘PasswordGenerator::CharClass’)`
so we have to replace all the calls with `static_casts`.

Reply:
Okay, I'm fine with leaving it as-is for now.